nrf_security/Kconfig.legacy

```diff
@@ -76,81 +76,102 @@ config MBEDTLS_AES_DECRYPT_ALT
 
 config MBEDTLS_AES_ALT
 	bool
-	depends on CC3XX_BACKEND || PSA_CRYPTO_DRIVER_CC3XX
+	depends on CC3XX_BACKEND || (PSA_CRYPTO_DRIVER_CC3XX || \
```

Comment:
Why ||? I can't see that this changes anything, as PSA_CRYPTO_DRIVER_HAS_CIPHER_SUPPORT_CC3XX depends on PSA_CRYPTO_DRIVER_CC3XX , so in effect only PSA_CRYPTO_DRIVER_CC3XX  is needed.

Was this supposed to be &&?

Reply:
Should be &&, will fi